### ISSUE TITLE
Does headless work?

### ISSUE THREAD

**Issue by singpolyma** [amonchakai/Hg10#93]
It seems like it might not, but it is hard to tell.

**Comment by dimitrisand** [comment]
Yes it does. I have a blackberry z10‎ and it works fine                                                                                                                                                                                                                                                                                                                                        Dimitris Andavoglou - Intec Systems S.A.                                                                                                                                                                                                                Από: Stephen Paul WeberΕστάλη: Τετάρτη, 12 Αυγούστου 2015 - 23:13Προς: amonchakai/Hg10Απάντηση σε: amonchakai/Hg10Θέμα: [Hg10] Does headless work? (#93)It seems like it might not, but it is hard to tell.

—Reply to this email directly or view it on GitHub.

**Comment by singpolyma** [comment]
I think I may just be getting disconnected, so the app is not getting messages reliably enough to test backgrounding or something.  I'm definitely getting no messages right now.
